python/tk_framework_desktopserver/shotgun/scripts/execute_command.py

```diff
@@ -382,18 +380,22 @@ def execute(
     arg_data_file = sys.argv[1]
 
     with open(arg_data_file, "rt") as fh:
-        arg_data = sgtk.util.json.load(fh)
+        arg_data = json.load(fh)
 
     # The RPC api has given us the path to its tk-core to prepend
-    # to our sys.path prior to importing sgtk. We'll prepent the
+    # to our sys.path prior to importing sgtk. We'll prepend the
     # the path, import sgtk, and then clean up after ourselves.
     original_sys_path = copy.copy(sys.path)
     try:
         sys.path = [arg_data["sys_path"]] + sys.path
         import sgtk
+        from tank_vendor import six
     finally:
         sys.path = original_sys_path
 
+    # Now that we have sgtk.util loaded, use it to make sure we have only utf-8 data in the args
+    arg_data = sgtk.util.unicode.ensure_contains_str(arg_data)
```

Comment:
We should have called sgtk.util.json.load(fh) the second time around. `unicode` module is private.